.circleci/config.yml

```diff
@@ -471,14 +477,35 @@ jobs:
       - store_artifacts:
           path: "test/integration/render-tests/index.html"
 
+  prepare-windows:
+    executor:
+      name: win/default
+    working_directory: ~/mapbox-gl-js
+    steps:
+      - checkout
+      - run: yarn --frozen-lockfile
+      - run: yarn run build-dev
+      - run:
+          name: Clean up workspace to persist faster
+          command: |
+            Remove-Item .git -Recurse -Force;
+            Remove-Item node_modules/flow-bin -Recurse -Force;
+            Remove-Item node_modules/tap -Recurse -Force;
+            Remove-Item node_modules/gl -Recurse -Force;
+            Remove-Item node_modules/@mapbox/mvt-fixtures/real-world/osm-qa-* -Recurse -Force;
+            Remove-Item node_modules/react-devtools-core -Recurse -Force;
+            Remove-Item node_modules/@octokit -Recurse -Force;
```

Comment:
Good call removing these unneeded directories! Could we be even more aggressive here in order to win the performance boost? Alternatively, could we provide a more limited set of directories to `persist_to_workspace`? (Maybe just `dist`, `package.json` and the dependencies needed to run tests)?

Reply:
That's the biggest offenders — I don't seem to be gaining much performance by going in more detail (and this becomes more fragile because e.g. it will need to be sometimes updated some deps are removed / added / updated).

Persisting inclusively could be worth a shot, although it might take a lot of lines to cover everything necessary (and same drawback as above — we'll have to maintain the list afterwards).